tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,338 @@
+        'releasever': u'6.7',
+        'label': u'rhel-6-server-rpms'
+    },
+    'rhel6_sat6tools': {
+        'product': u'Red Hat Enterprise Linux Server',
+        'reposet': u'Red Hat Satellite Tools 6.1 (for RHEL 6 Server) '
+                   '(RPMs)',
+        'reponame': u'Red Hat Satellite Tools 6.1 for RHEL 6 Server RPMs '
+                    'x86_64',
+        'arch': u'x86_64',
+        'releasever': None,
+        'label': u'rhel-6-server-satellite-tools-6.1-rpms'
+    }
+}
+
+
+@run_only_on('sat')
+class TestIncrementalUpdate(TestCase):
+    """Tests for the Incremental Update feature"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Creates all the pre-requisites for the Incremental updates test"""
+        # Step 1 - Create a new Organization
+        cls.org = Organization(name=gen_alpha(length=5)).create()
+
+        # Step 2 - Create two life cycle environments - DEV, QE
+        cls.dev_lce = LifecycleEnvironment(
+            name='DEV',
+            organization=cls.org
+        ).create()
+        cls.qe_lce = LifecycleEnvironment(
+            name='QE', prior=cls.dev_lce, organization=cls.org
+        ).create()
+
+        # Step 3: Upload manifest
+        with open(manifests.clone(), 'rb') as manifest:
+            upload_manifest(cls.org.id, manifest)
+
+        # Step 4: Enable repositories - 6Server and rhel6 sat6tools
+        rhel_66_repo_id = enable_rhrepo_and_fetchid(
+            basearch=PRD_SETS['rhel_66']['arch'],
+            org_id=cls.org.id,
+            product=PRD_SETS['rhel_66']['product'],
+            repo=PRD_SETS['rhel_66']['reponame'],
+            reposet=PRD_SETS['rhel_66']['reposet'],
+            releasever=PRD_SETS['rhel_66']['releasever']
+        )
+        rhel6_sat6tools_repo_id = enable_rhrepo_and_fetchid(
+            basearch=PRD_SETS['rhel6_sat6tools']['arch'],
+            org_id=cls.org.id,
+            product=PRD_SETS['rhel6_sat6tools']['product'],
+            repo=PRD_SETS['rhel6_sat6tools']['reponame'],
+            reposet=PRD_SETS['rhel6_sat6tools']['reposet'],
+            releasever=PRD_SETS['rhel6_sat6tools']['releasever']
+        )
+
+        # Step 5: Read the repositories
+        cls.rhel_66_repo = Repository(id=rhel_66_repo_id).read()
+        cls.rhel6_sat6tools_repo = Repository(
+            id=rhel6_sat6tools_repo_id
+        ).read()
+
+        # Step 6: Sync the enabled repositories
+        try:
+            cls.old_task_timeout = entity_mixins.TASK_TIMEOUT
+            # Update timeout to 2 hours to finish sync
+            entity_mixins.TASK_TIMEOUT = 7200
+            for repo in [cls.rhel_66_repo, cls.rhel6_sat6tools_repo]:
+                response = Repository(id=repo.id).sync()
+                assert response['result'] == u'success'
+        finally:
+            entity_mixins.TASK_TIMEOUT = cls.old_task_timeout
+
+        # Step 7: Create two content views - one will be used with all erratas
+        # and another with erratas filtered
```

Comment:
Two content views are put to test - one will have all errata in it by default.  The other would have fitered out errata which were delivered in the last 1 year.